{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "40789413",
  "multipleFaultNumbers": null,
  "date": "2020-12-05T22:19:08",
  "region": "Bolton",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 1,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-05T22:19:08",
  "information": "",
  "actualTimeOfRestoration": "2020-12-05T22:19:08",
  "outageCentrePoint": {
    "lat": 53.545052,
    "lng": -2.425629
  },
  "outageLocations": null,
  "AffectedPostcodes": " BL4 0QX, BL4 0RA, BL4 0RB, BL4 0RD, BL4 0RE, BL4 0RF, BL4 0RG, BL4 0RH, BL4 0RQ",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}